src/Altinn.Platform/Altinn.Platform.Profile/Profile/Startup.cs

```diff
@@ -173,6 +175,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapControllers();
+                endpoints.MapHealthChecks("/health");
```

Comment:
This endpoint mapping is missing from the other platform components. Should it be included for the others or removed from here?

Reply:
Added.